docs/releasing.md

```diff
@@ -10,7 +67,14 @@ Releasing
     export NEXT_VERSION=X.Y.Z-SNAPSHOT
```

Comment:
just wanted to make sure. is the example of this like below? 👀 
```
export RELEASE_VERSION=4.0.2
export NEXT_VERSION=4.0.3-SNAPSHOT
```

Reply:
yep!